python/ray/actor.py

```diff
@@ -163,6 +163,7 @@ def remote(self, *args, **kwargs):
 
         return FuncWrapper()
 
+    @wrap_auto_init
```

Comment:
auto-init needs to happen before the tracing decorator runs because it calls `get_runtime_context`